### PR TITLE
feat(angular): update @angular-eslint to v1

### LINKED ISSUE #4294: NX11 does not work with double qoutes
## Current Behavior
When using double quotes in `.prettierrc` and run nx lint, nx is complaining about wrong quotes.

```
1:31  error  Strings must use singlequote  quotes
2:26  error  Strings must use singlequote  quotes
4:30  error  Strings must use singlequote  quotes
```

## Expected Behavior
Nx lint shouldn't show error for single quotes when project is set to double quotes as in nx <11

## Steps to Reproduce
* Create a new NX11 angular project with eslint and angular
* Change `singleQuote` in `.prettierrc` to false
* Run `nx format:write`
* Run `nx affected:lint`
* See lint errors

I noticed that the eslint config inside apps and libs was changed from `nx 10` to `nx 11`

```
{ "extends": "../../.eslintrc.json", "ignorePatterns": ["!**/*"], "rules": {} }
```

```
{
  "extends": ["../../.eslintrc.json"],
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "extends": [
        "plugin:@nrwl/nx/angular",
        "plugin:@angular-eslint/template/process-inline-templates"
      ],
      "parserOptions": {
        "project": ["apps/other-angular-app/tsconfig.*?.json"]
      },
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          { "type": "attribute", "prefix": "app", "style": "camelCase" }
        ],
        "@angular-eslint/component-selector": [
          "error",
          { "type": "element", "prefix": "app", "style": "kebab-case" }
        ]
      }
    },
    {
      "files": ["*.html"],
      "extends": ["plugin:@nrwl/nx/angular-template"],
      "rules": {}
    }
  ]
}

```

### Environment

```
  Node : 12.18.3
  OS   : darwin x64
  npm  : 6.14.8
  
  nx : Not Found
  @nrwl/angular : 11.0.2
  @nrwl/cli : 11.0.2
  @nrwl/cypress : 11.0.2
  @nrwl/devkit : 11.0.2
  @nrwl/eslint-plugin-nx : 11.0.2
  @nrwl/express : Not Found
  @nrwl/jest : 11.0.2
  @nrwl/linter : 11.0.2
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.2
  @nrwl/web : Not Found
  @nrwl/workspace : 11.0.2
  typescript : 4.0.5
```

### PR DESCRIPTION
**Pull request by JamesHenry**
I released @angular-eslint v1 yesterday and so updating here.

Part of the update to v1 removed all stylistic rules from the `recommended` config so it also:

Fixes #4294

### CODE REVIEW

File: packages/angular/migrations.json [hunk 1]
```diff
@@ -262,6 +267,23 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "11.1.0": {
+      "version": "11.1.0-beta.0",
```
Comment (JamesHenry) [inline packages/angular/migrations.json:272]: I guessed at this being a good upcoming point for this

File: packages/linter/src/builders/eslint/lint.impl.spec.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ async function runBuilder(options: Schema) {
   registry.addPostTransform(schema.transforms.addUndefinedDefaults);
 
   const testArchitectHost = new TestingArchitectHost('/root', '/root');
-  const builderName = '@angular-eslint/builder:lint';
+  const builderName = '@nrwl/linter:eslint';
```
Comment (JamesHenry) [inline packages/linter/src/builders/eslint/lint.impl.spec.ts:69]: Fixed and old copy paste issue

File: packages/linter/src/builders/lint/lint.impl.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import { createDirectory } from '@nrwl/workspace';
 async function run(options: Schema, context: BuilderContext): Promise<any> {
   if (options.linter === 'tslint') {
     throw new Error(
-      `'tslint' option is no longer supported. Update your angular.json to use "@angular-eslint/builder:lint" builder directly.`
+      `'tslint' option is no longer supported. Update your angular.json to use "@nrwl/linter:eslint" builder.`
```
Comment (JamesHenry) [inline packages/linter/src/builders/lint/lint.impl.ts:16]: This text should have been updated when the builder was changed so fixed it as well

File: packages/angular/migrations.json (hunk 1 above)
Comment (vsavkin) [reply packages/angular/migrations.json:272]: Yup. This is perfect

**Pull request merged**